src/executor/current_thread/mod.rs

```diff
@@ -147,9 +147,10 @@ pub struct TaskExecutor {
     _p: ::std::marker::PhantomData<Rc<()>>,
 }
 
-/// Returned by the `turn` function
+/// Returned by the `turn` function. Contains `true` if any futures were
```

Comment:
Tiny nit. Could you move the second sentence to a new paragraph (rust doc formatting).